signing/signer.js

```diff
@@ -12,7 +12,7 @@ export async function getSigner(signingType, { address, password }, chainId) {
     return wallet
   } else if (signingType === `ledger`) {
     return await getLedger()
-  } else if (signingType === `extension`) {
+  } else if (signingType === `keplr`) {
```

Comment:
this is going to work for lunie ext and keplr?

Reply:
no, lunie extension is using a different logic